tests/test_machine.py

```diff
@@ -0,0 +1,65 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+import os
+from subprocess import TimeoutExpired
+
+import pytest
+
+import mkosi.machine as machine
+from mkosi.backend import MkosiException
+
+
+pytestmark = [pytest.mark.integration, pytest.mark.parametrize("verb", ["boot", "qemu"]), pytest.mark.skipif(os.getuid() != 0, reason="Must be invoked as root.")]
```

Comment:
For better leggibility
```suggestion
pytestmark = [
    pytest.mark.integration, 
    pytest.mark.parametrize("verb", ["boot", "qemu"]), 
    pytest.mark.skipif(os.getuid() != 0, reason="Must be invoked as root.")
]
```